daycare/migrations/0003_rename_image_url_image_url.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.0.3 on 2022-04-11 09:47
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('daycare', '0002_test_data'),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name='image',
+            old_name='image_url',
+            new_name='url',
```

Comment:
Instead of having several files of the changes in the model, you can delete all the auto-generated migrations, and make Django create a new one for you, in a single file.
As for the 2nd file, for the data_test, just move it to a different directory, and then start the "makemigrations" command.

Reply:
Take a look at the comment above of jenia and omer, about changing the migrations files. In that case for each new update in the data migrations, the old migrations shouldn't be changed.

Reply:
Don't worry too much about having multiple migration files, they don't weigh a lot. :) 
Migrations should not be changed manually; that means no modifications and no deletion. a migration created is a migration to stay. info is either modified or removed by new migrations. 
The only exception to the rule is if you created multiple migrations, but have yet to post this change. you can remove them and let django re-create it. but take into account that you'll need to reset your local db accordingly.